src/skai/model/xmanager_helper_module.py

```diff
@@ -0,0 +1,349 @@
+"""
+Code in this module is adapted from the XManager repo as a 
+temporary workaround for TypeError when importing xm from xmanager in python 3.8.
+
+Code source: https://github.com/deepmind/xmanager/tree/main/xmanager/vizier/vizier_cloud
+"""
+
+import re
+import time
+import abc
+from absl import logging
+from xmanager.cloud import auth
+from typing import Any, Dict, Optional
+from google.cloud import aiplatform_v1beta1 as aip
+
+_DEFAULT_LOCATION = 'us-central1'
```

Comment:
Let's not make any hidden assumptions about the user's preferred location. It may lead to hard-to-find performance problems that the user is not aware of. Instead, let's just force the user to always pass in the cloud location.